MovieHubRx/Scenes/Favorite/FavoriteNavigator.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FavoriteNavigator.swift
+//  MovieHubRx
+//
+//  Created by Duy Nguyễn on 13/05/2024.
+//
+
+import Foundation
+import UIKit
+import RxSwift
+import RxCocoa
+
+protocol FavoriteNavigatorType {
+    
+}
+
+struct FavoriteNavigator: FavoriteNavigatorType {
+    var navigationController = UINavigationController()
```

Comment:
nên dùng `unowned` tránh retain cycle e
```suggestion
    unowned let navigationController = UINavigationController()
```